lib/v2/npr/full.js

```diff
@@ -0,0 +1,61 @@
+const parser = require('@/utils/rss-parser');
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const getArticleDetail = (link, ctx) =>
+    ctx.cache.tryGet(link, async () => {
+        const response = await got(link);
+        const $ = cheerio.load(response.data);
+
+        const categories = $('.slug').text().trim();
+
+        const audioLink = $('.audio-tool-download a').attr('href');
+        const audio = audioLink ? `<audio controls><source src="${audioLink}" type="audio/mp3"></audio>` : '';
+
+        const regex = /\?storyId=(\d+)&amp;mediaId=(\d+)/;
+        const m = $.html().match(regex);
+        let video = '';
+        if (m) {
+            const storyId = m[1];
+            const mediaId = m[2];
+            video = $('.npr-video').length ? `<iframe width="740" height="416" src="https://www.npr.org/embedded-video?storyId=${storyId}&mediaId=${mediaId}&jwMediaType=music" frameborder="0" scrolling="no"></iframe>` : '';
+        }
+
+        const article =
+            audio +
+            video +
+            Array.from(
+                $('.storytext')
+                    .remove('.bucketwrap ')
+                    .map((i, el) => $(el).html())
+            ).join('');
```

Comment:
`.bucketwrap` didn't get removed from the article.
https://github.com/DIYgod/RSSHub/pull/12827#issuecomment-1639546061

It would be better to leave `.bucketwrap` untouched and remove other elements like `.enlarge-options, .toggle-caption, .hide-caption, .enlarge_measure, .enlarge_html, .internallink` and `aside[aria-label="advertisement"]` instead.